src/codex_plus/llm_execution_middleware.py

```
@@ -326,46 +351,65 @@ async def process_request(self, request, path: str):
-                            # ✅ SAFE TO MODIFY: Status line injection customization
-                            if not status_line_injected and status_line:
-                                # Format like official Claude Code status line (simple, clean format)
-                                status_content = f"{status_line}\n\n"
-                                yield status_content.encode('utf-8')
-                                status_line_injected = True
-                                logger.info(f"✅ Status line injected into stream: {status_line}")
-
                             # 🔒 PROTECTED: Chunk yielding - DO NOT REMOVE
                             yield chunk
                 except Exception as e:
                     logger.error(f"Error during streaming: {e}")
```

Ruff (0.13.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)